charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -97,6 +101,10 @@ spec:
             - name: scalardb-cluster-tls-volume
               mountPath: /tls/scalardb-cluster/certs
             {{- end }}
+            {{- if and (.Values.scalardbCluster.tde.enabled) (eq .Values.scalardbCluster.tde.type "vault") (.Values.scalardbCluster.tde.vault.tls.enabled) }}
+            - name: scalardb-cluster-tde-vault-tls-volume
+              mountPath: /tde/vault/tls/certs
+            {{- end }}
```

Comment:
If we use the `vault` type TDE and use TLS between ScalarDB Cluster and Vault, we can set a CA root certificate to access to Vault from ScalarDB Cluster by using a secret resource. This chart mounts the CA root certificate on pods from the secret resource.